### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by brandonaaskov** [peers/peerjs#360]
I'm looking to use a WebRTC library for an upcoming project, and looking at the recent issues, and not-very-recent commits, I'm left wondering... is this abandonware?

**Comment by digitalgarage** [comment]
Hi @brandonaaskov . I can't answer on the status of the project.  However I have successfully used peerjs on a couple of projects.

Bear in mind that Chrome & possibly FF will both require ssl connections to access the mediastream which means that rather than using peerjs cloud you will need to host your own peerjs server.  But that is very, very easy with heroku and the one click peerjs server install.

Secondly I have found connections a little hit and miss when I haven't used stun and turn servers.  I ended up using xirsys.com to provide that and it's worked great!

Just remember that IE and Safari still don't support the WebRTC spec :(

EF.
